src/nvim/eval.lua

```diff
@@ -4475,6 +4475,7 @@ M.funcs = {
     ]=],
     name = 'getscriptinfo',
     params = { { 'opts', 'table' } },
+    returns = 'vim.fn.getscriptinfo.ret|vim.fn.getscriptinfo.ret[]',
```

Comment:
This function always returns a list

Reply:
Nope, the docs say that is `sid` is specified, only information about the script with ID `"sid"` is returned. There is even this example: `echo getscriptinfo({'sid': 15}).variables`

Reply:
Hmm, the docs is wrong. When `sid` specified, it returns a list with a single item.

Reply:
Oh yes you're right, I just checked the tests and it always returns a list. Fixing it.